### PR TITLE
Update for transfer to mamba-org

### PR DESCRIPTION
**Pull request by eitsupi**
Related to #8

As discussed in mamba-org/micromamba-devcontainer#7, I have transferred this repository to mamba-org and have updated some documentation.

Discussion: I think it would be a good idea to take this opportunity to update the version to 1.0.0.

### CODE REVIEW

File: src/micromamba/README.md [hunk 1]
```diff
@@ -1,46 +0,0 @@
-
```
Comment (wolfv) [inline src/micromamba/README.md:1]: did you want to move this file to the root README?

**Comment by wolfv** [comment]
Looks good to me! Just confused about the removal of the README file (I think the current README is pretty empty as well).

File: src/micromamba/README.md (hunk 1 above)
Comment (eitsupi) [reply src/micromamba/README.md:1]: Thank you for your review.

Sorry for the lack of explanation, the README for the Feature has been temporarily removed as it is automatically generated by the GitHub Actions release workflow.
A new README will be generated when the release workflow is performed.

Comment (wolfv) [reply src/micromamba/README.md:1]: great.

**Review by maresb — APPROVED** [review]
I read through everything again, and I extended the base README. Everything looks great, thanks a lot! Looks ready to merge.

**Comment by eitsupi** [comment]
@maresb Thanks for your review!
Do you think the version should remain 0.1.0? I think it is worth updating to version 1.0.0.

**Comment by maresb** [comment]
> I think it is worth updating to version 1.0.0.

I agree, can you do that and merge? Then once https://github.com/devcontainers/devcontainers.github.io/pull/118 comes out of draft I will give an approving review.

**Comment by eitsupi** [comment]
Released👍
image

**Pull request merged**